### ISSUE TITLE
bot.send_photo() Throws Errors

### ISSUE THREAD

**Issue by i-bond** [eternnoir/pyTelegramBotAPI#940]
Please answer these questions before submitting your issue. Thanks!
1. What version of pyTelegramBotAPI are you using? 
 3.7.2
2. What OS are you using? 
Windows 
3. What version of python are you using?
3.7

Hello! I encountered the following problem, every time when sending a photo,  my bot throws the same error, here is the traceback:
```
Traceback (most recent call last):
  File "C:/Users/User/Desktop/Projects/bot-telegram/bot.py", line 2144, in <module>
    bot.send_photo(355146804, photo='https://drive.google.com/file/d/1bmOtP5W30ILW43cY63VUPBN6kxdM0ka_/view?usp=sharing')
  File "C:\Users\User\Desktop\Projects\bot-telegram\venv\lib\site-packages\telebot\__init__.py", line 677, in send_photo
    parse_mode, disable_notification))
  File "C:\Users\User\Desktop\Projects\bot-telegram\venv\lib\site-packages\telebot\apihelper.py", line 295, in send_photo
    return _make_request(token, method_url, params=payload, files=files, method='post')
  File "C:\Users\User\Desktop\Projects\bot-telegram\venv\lib\site-packages\telebot\apihelper.py", line 65, in _make_request
    return _check_result(method_name, result)['result']
  File "C:\Users\User\Desktop\Projects\bot-telegram\venv\lib\site-packages\telebot\apihelper.py", line 84, in _check_result
    raise ApiException(msg, method_name, result)
telebot.apihelper.ApiException: A request to the Telegram API was unsuccessful. The server returned HTTP 400 Bad Request. Response body:
[b'{"ok":false,"error_code":400,"description":"Bad Request: wrong file identifier/HTTP URL specified"}']
```

For some reasone it works with this image:
`bot.send_photo(your_id, photo='https://drive.google.com/file/d/10Ez-WV5_55Kc2MoMPA88elkKtIBWRj6D/view?usp=sharing')`
And always fails when it's changed to:
`bot.send_photo(your_id, photo='https://drive.google.com/file/d/1bmOtP5W30ILW43cY63VUPBN6kxdM0ka_/view?usp=sharing')`

Could you help me to find out what causes this error?

**Comment by mrpes** [comment]
Second link is not public

**Comment by i-bond** [comment]
Changed the link to public, but it still doesn't work (

**Comment by mrpes** [comment]
Well this is strange. I downloaded both images and tried to send them from local, and it worked. The only thing i noticed is that one image is .jpg while the other is .png. But they even not corrupted. Maybe telegram just wont work with .png uploaded on google docs?.. Because I tried to upload .png from other site, and again all worked fine. I suggest you try to convert your `welcome.png` to jpg, or use another image hosting solution
